entity-framework/core/miscellaneous/1x-2x-upgrade.md

```diff
@@ -31,9 +31,9 @@ Updating an existing application to EF Core 2.0 may require:
 
 ## Breaking Changes
 
-We have taken the opportunity to significantly refine our existing APIs and behaviors in 2.0. There are a few improvements that can require modifying existing application code, although we believe that for the majority of applications the impact will be low, in most cases requiring just recompilation and minimal guided changes to replace obsolete APIs.
+We have taken the opportunity to significantly refine our existing APIs and behaviors in 2.0. There are a few improvements that can require modifying existing application code, although we believe that for the majority of applications the impact will be low, in most cases requiring just recompilation and minimal guided changes to replace obsolete APIs. The following sections provide details about breaking changes.
 
-### New way of getting application services
+## New way of getting application services
```

Comment:
For this particular page, I can see that everything that you need to change when you upgrade to 2.0 is by definition because of a breaking change, so I can imagine us just reorganizing the page to remove the "Breaking changes" title altogether and moving the paragraph that explains that we have made changes to the top. Sounds ok? I can try to explain more if needed.